ui/src/components/Views/Update/UpdateDialog.tsx

```diff
@@ -51,16 +56,29 @@ export const UpdateDialog = ({ open, onClose }: Props): ReactElement => {
     <Dialog open={open} onClose={onClose}>
       <DialogContent>
         <Box m={2} width={500} height={400}>
-          {logs.map(log => (
+          {
+            logs.map(log => (
+              <>
+                <Typography>
+                  {log}
+                </Typography>
+                <br />
+              </>
+            ))
+          }
+          {
+            logs.length === 0 &&
```

Comment:
Once you launch the `localstack update` command from the cli "Updating docker images" is the first string you get but it takes some seconds to display it, if I checked this when `isUpdating` we would have it duplicated since I', not filtering the output of the command